shared/unlock-folders/device-list.desktop.js

```diff
@@ -29,23 +39,26 @@ export default class DeviceList extends Component<void, Props, void> {
   render () {
     return (
       <div style={{...globalStyles.flexBoxColumn, alignItems: 'center'}}>
-        <Text type='Body' style={styles.infoText}>Turn on one of your devices to unlock your folders:</Text>
-        <div style={styles.devicesContainer}>
-          {this.props.devices && this.props.devices.map(d => <DeviceRow key={d.deviceID} device={d} />)}
+        <Text type='Body' style={styles.infoText}>This computer and possibly others are unable to read some of your folders. To avoid losing data forever, please turn on one of the devices below:</Text>
```

Comment:
This text is from coyne which matches closer to what we show in the files screen